__tests__/utils.test.js

```diff
@@ -102,3 +103,148 @@ describe("formatComments", () => {
+        created_at: 1604394720000,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      },
+    ];
+    const inputTwo = "body";
+    const expected = [
+      {
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        created_at: 1594329060000,
+        votes: 100,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      },
+      {
+        title: "Sony Vaio; or, The Laptop",
+        topic: "mitch",
+        author: "icellusedkars",
+        created_at: 1602828180000,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      },
+      {
+        title: "Eight pug gifs that remind me of mitch",
+        topic: "mitch",
+        author: "icellusedkars",
+        created_at: 1604394720000,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      },
+    ];
+    const actual = removePropertyFromObjectArray(inputOne, inputTwo);
+    expect(actual).toEqual(expected);
+  });
+  it("should not mutate the input array", () => {
+    const inputOne = [
+      {
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: 1594329060000,
+        votes: 100,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      },
+    ];
+    const inputOneCopy = [
+      {
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: 1594329060000,
+        votes: 100,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      },
+    ];
+    const inputTwo = "body";
+    removePropertyFromObjectArray(inputOne, inputTwo);
+    expect(inputOne).toEqual(inputOneCopy);
+  });
+  it("should have an ouput with a different memory reference to that of the input", () => {
+    const inputOne = [
+      {
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: 1594329060000,
+        votes: 100,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      },
+    ];
+    const inputTwo = "body";
+    const actual = removePropertyFromObjectArray(inputOne, inputTwo);
+    expect(actual).not.toBe(inputOne)
```

Comment:
Clever, but we mustn't remove the body from our articles using JavaScript, we must edit our SQL query to get all the columns except the body. Please make sure you are doing this 🙂